examples/pigweed-app/esp32/include/AppConfig.h

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+// ESP32 Logging
+// ---- PW Example App Config ----
+#include "esp_log.h"
+#include "PigweedLoggerMutex.h"
+#include <support/logging/CHIPLogging.h>
+
+extern "C" void __wrap_esp_log_write(esp_log_level_t level, const char *tag, const char *format, ...)
+{
+    va_list v;
+    va_start(v, format);
+#if CONFIG_LOG_DEFAULT_LEVEL > 1
+    if(strcmp(tag, "spi_flash") != 0 && strcmp(tag, "cpu_start") != 0 )
```

Comment:
What's the reason for filtering out these tags?

Reply:
ESP_LOG* are sent to UART0. This UART must be initialized first. It gets initialized after the CPU starts. For `spi_flash` and `cpu_start` logs UART remains uninitialised. Hence they were filtered out. And these logs are not that important to consume buffer for them hence they are ignored.